deploy-aks/azext_aks_deploy/dev/aks/up.py

```diff
@@ -29,7 +33,9 @@ def aks_deploy(aks_cluster=None, acr=None, repository=None, port=None, skip_secr
     :type repository: str
     :param port: Port on which your application runs. Default is 8080
     :type port:str
-    :param skip_secrets_generation : Flag to skip generating Azure credentials.
+    :param branch_name: New branch name to be created for checking files and raise a PR
```

Comment:
Wording should be "to check in" rather than "for checking". 

Might need to be more elaborate for behavior in case of skipping this param. Will it start asking for this interactively? 
How can I automate direct checkin rather than PR creation?

Reply:
Updated the wording.

Great point regarding automation scenario!
@geverghe, what should be the expected behavior in case of automation where user wants to commit to default branch ?
Also, are we even targeting these commands for automation ?
Or just the interactive flow